src/components/PasswordInput/index.js

```diff
@@ -0,0 +1,5 @@
+import PasswordInput from './PasswordInput';
+
+export { PasswordInput };
+
+export default PasswordInput;
```

Comment:
Why both if they are the same? For consistency?

Reply:
Yeah, I'm a bit confused about this. Usually they shouldn't be the same and this actually causes eslint errors when you import the default prop with the same name as the named export. For example,

```javascript
import PasswordInput from '../PasswordInput'
```

Usually the default export should have some HOCs, but we don't have them... Should we go for only default exports when there are no HOCs? As a convention? Once or something similar get involved, we could make the wrapped component the default export and move the plain one to named export?